jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/solver/stats/Stats.java

```diff
@@ -103,7 +105,7 @@ private static Item format(Map<Node, Integer> predicates, Map<Node, Integer> typ
         addPair(meta.getList(), "timestamp", NodeFactoryExtra.nowAsDateTime());
         addPair(meta.getList(), "run@",  DateTimeUtils.nowAsString());
         if ( count >= 0 )
-            addPair(meta.getList(), StatsMatcher.COUNT, NodeFactoryExtra.intToNode((int)count));
```

Comment:
The problem is the cast.

`NodeFactoryExtra.intToNode(count)` should work.  `intToNode(long integer)` exists.

They both return an xsd:integer which is a better choice than xsd:long because, if nothing else, it prints nicely.